espflash/src/targets/esp32.rs

```diff
@@ -157,12 +155,12 @@ impl Target for Esp32 {
 
     fn flash_image<'a>(
         &self,
-        elf: ElfFile<'a>,
+        elf_data: &'a [u8],
         flash_data: FlashData,
         _chip_revision: Option<(u32, u32)>,
         xtal_freq: XtalFrequency,
     ) -> Result<IdfBootloaderFormat<'a>, Error> {
-        let booloader: &'static [u8] = match xtal_freq {
```

Comment:
https://github.com/esp-rs/esp-hal/pull/3281#discussion_r2018904119 funny that we found the same typo in two different projects in the same day